docs/spark-standalone.md

```diff
@@ -260,6 +261,17 @@ SPARK_WORKER_OPTS supports the following system properties:
     especially if you run jobs very frequently.
   </td>
 </tr>
+<tr>
+  <td><spark.shuffle.service.db.enabled</code></td>
+  <td>true</td>
+  <td>
+    Enable record RegisteredExecutors information by leveldb, which can be reloaded and
+    used again when the external shuffle service is restarted. Note that this only affects standalone
+    mode, its has always on for yarn. We should Enable `spark.worker.cleanup.enabled` to remove the entry
+    (It will leave an entry in the DB forever when an application is stopped while the external shuffle
+    service is down) in the leveldb with WorkDirCleanup. It may be removed in the future.
```

Comment:
Some minor rewordings:

```
Store External Shuffle service state on local disk so that when the external shuffle service is restarted, it will
automatically reload info on current executors.  This only affects standalone mode (yarn always has this behavior
enabled).  You should also enable <code>spark.worker.cleanup.enabled</code>, to ensure that the state
eventually gets cleaned up.  This config may be removed in the future.
```

Reply:
Very clear description, thank you.